### PR TITLE
[Collectible] check coins in collectible api

### PR DESCRIPTION
**Pull request by hewigovens**
also update golibs/coin package

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -216,10 +216,10 @@ filecoin:
   api: https://api.filscan.io:8700/rpc/v1
   explorer: https://filfox.info
 
-# smartchain:
-#   api: <blockbook api>
-#   rpc: https://bsc-dataseed.nariox.org
-#   collections_api: https://nftview.bounce.finance
+smartchain:
+  api: <blockbook api>
+  rpc: https://bsc-dataseed.binance.org
```
Comment (vikmeup) [inline config.yml:221]: `smartchain.rpc` is this still required url? I don't think we ever use it in the blockatlas, same for ethereum

File: platform/harmony/transaction.go [hunk 2]
```diff
@@ -64,7 +64,7 @@ func NormalizeTx(trx *Transaction) (tx types.Tx, b bool, err error) {
 
 	return types.Tx{
 		ID:       trx.Hash,
-		Coin:     coin.ONE,
+		Coin:     coin.HARMONY,
```
Comment (vikmeup) [inline platform/harmony/transaction.go:67]:
```suggestion
		Coin:     coin.HARMONY,
```

maybe we use `coin.Harmony().ID` for consistency?

**Review by vikmeup — APPROVED** [review]

File: platform/harmony/transaction.go (hunk 2 above)
Comment (hewigovens) [reply platform/harmony/transaction.go:67]: I'd like keep direct accessing `coin.HARMONY`, unlike `name`, `symbol` it's defined in the map, less call

File: config.yml (hunk 1 above)
Comment (hewigovens) [reply config.yml:221]: ok, let me remove it

Comment (hewigovens) [reply config.yml]: done

**Pull request merged**